### PR TITLE
Add 'nat' as en enable flag

### LINKED ISSUE #69: Make NAT a disableable tunable

### PR DESCRIPTION
**Pull request by gozer**
Warning: pretty much everythign breaks if it's not enabled.

Fixes #69

### CODE REVIEW

File: input.tf [hunk 1]
```diff
@@ -49,13 +49,15 @@ variable technical_contact {
 
 variable features {
   default = {
-    consul       = 0
+    consul       = 1
```
Comment (gozer) [inline input.tf]: Saner default, IMO

File: input.tf [hunk 2]
```diff
@@ -49,13 +49,15 @@ variable technical_contact {
...
     jumphost     = 0
     fluent       = 0
     stack_compat = 0
     mig          = 0
     ci           = 0
     vpn          = 0
+    nat          = 1
```
Comment (gozer) [inline input.tf]: Saner default, IMO

File: input.tf [hunk 3]
```diff
@@ -49,13 +49,15 @@ variable technical_contact {
...
+    opsec        = 0
```
Comment (gozer) [inline input.tf]: Just adding for documentation, got missed

**Comment by limed** [comment]
r+

**Pull request merged**